packages/app-desktop/gui/Sidebar/style.scss

```
@@ -1,9 +1,10 @@
-@use 'styles/folder-and-tag-list.scss';
-@use 'styles/list-item-wrapper.scss';
-@use 'styles/note-count-label.scss';
-@use 'styles/sidebar-expand-icon.scss';
-@use 'styles/sidebar-expand-link.scss';
-@use 'styles/sidebar-header-container.scss';
-@use 'styles/sidebar-spacer-item.scss';
-@use 'styles/sidebar-header-button.scss';
-@use 'styles/sidebar-sync-button.scss';
\ No newline at end of file
+@use "styles/folder-and-tag-list.scss";
```

Stylelint (17.3.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)